apps/system/js/call_forwarding.js

```diff
@@ -100,12 +99,9 @@
         if (value === null) {
```

Comment:
Note that in `_initCallForwardingState` we always need to enable/disable the icon (using `_callForwardingHelper`) based on the card state and the value store under `'ril.cf.enabled.' + iccid` because we no longer reset the icons in the first place. The only exception that we do an early return would be `_callForwardingIconInitializedStates[index]` is true, and this happens only when the icon has been enabled.